pyscriptjs/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "_version": "3.0.0",
 
     "include": ["src/**/*"],
-    "exclude": ["node_modules/*", "__sapper__/*", "public/*"],
+    "exclude": ["node_modules/*", "__sapper__/*", "public/*", "src/interpreter_worker/*"],
```

Comment:
Why do you exclude this? You have some type errors currently?

Reply:
add to the top of the file:
```ts
declare function importScripts(src: string | string[]): void;
```

Reply:
I guess you can also add:
```ts
/// <reference lib="WebWorker" />
```
to the top of the file and it should work.

Reply:
Not really, no. It's easier to have one tsconfig for everthing.

Reply:
I'm at the end of my day, but feel free to give it a try if you want :)

Reply:
I'm also okay handling these issues in followups.